practice-one/src/helpers/validation.ts

```diff
@@ -0,0 +1,23 @@
+import REGEX from "../constants/regex";
+
+export const isValidName = (name: string) => {
```

Comment:
just use one function for both case
export const isValidEmail = (email: string, type) => {
    const regex = REGEX[type];
    return regex.test(type);
};